appstats.js

```diff
@@ -104,9 +104,13 @@ class AzureWebAppStats {
             contToken.token,
             function(error, result) {
                 if (error) {
-                    obj[functionName] = {
-                        error : `${error}`
-                    };
+                    if(process.env.FUNCTIONS_EXTENSION_VERSION === '~4'){
+                        obj[functionName] = appstats;
```

Comment:
should we rather have empty stats object here instead of `appStats`:
```
{
            invocations : 0,
            errors : 0
        };
```

Reply:
resolved